corehq/apps/repeaters/models.py

```diff
@@ -50,7 +50,7 @@
 def simple_post_with_cached_timeout(data, url, expiry=60 * 60, force_send=False, *args, **kwargs):
     # no control characters (e.g. '/') in keys
     key = hashlib.md5(
-        '{0} timeout {1}'.format(__name__, url)
+        u'{0} timeout {1} {2}'.format(__name__, url, data).encode('utf-8')
```

Comment:
Our guess is there's a chance that this just won't be an issue now with all the significant changes you mentioned @benrudolph